clients/include/flops.hpp

```diff
@@ -93,6 +93,25 @@ constexpr double scal_gflop_count<rocblas_double_complex, double>(rocblas_int n)
  * ===========================================================================
  */
 
+/* \brief floating point counts of trmv */
+template <typename T>
+constexpr double trmv_gflop_count(rocblas_int m)
+{
+    return ((m * (m + 1))) / 1e9;
+}
+
+template <>
+constexpr double trmv_gflop_count<rocblas_float_complex>(rocblas_int m)
+{
+    return (2.0 * (m * (m + 1))) / 1e9;
```

Comment:
According to http://www.netlib.org/blas/cblas2time.f this should be 4 x m x m

Reply:
okay ... but if I count the operation (out of diagonal non unit or unit) should be m(m+1)/2 instead of m*m.
As a result, 2 * m * (m+1).

Reply:
There are m(m+1)/2 complex multiplications, plus the same amount of complex additions. Each complex multiplication is 6 operations (4 multiplies and 2 adds) and each complex addition is 2 operations. This would be:
3m(m+1) + m(m+1) = 4m(m+1). I might be overlooking something though.

On netlib, _real_ gemv is calculated to have 2m*n + 2m operations, this is because of the multiply and the add, although I'm not sure where the extra 2m is coming from.

Reply:
Extra 2m is coming from operations on y.

Okay, so (m(m+1)/2 multiplications + (m(m+1))/2 additions, means that for the float double we are good. (Actually, it should be (m(m+1)/2) multiplications  + ((m-1)*m)/2 which would lead to m*m operations)

For the complex case, you are right the constant must be 4.
(or 4*(m(m+1)/2)+2*(m(m-1))/2) + 2*(m(m-1))/2  = 2m(m-1) + 2m(m+1) = 2m(2m) = 4*m*m.

So m*m and 4*m*m for complex types.
Thank you Dayne, it is clear now.